samcli/lib/build/app_builder.py

```diff
@@ -611,3 +629,73 @@ def _parse_builder_response(stdout_data: str, image_name: str) -> Dict:
             raise ValueError(msg)
 
         return cast(Dict, response)
+
+    @staticmethod
+    def _get_env_vars_value(filename: Optional[str]) -> Any:
+        """
+        If the user provided a file containing values of environment variables, this method will read the file and
+        return its value
+
+        :param string filename: Path to file containing environment variable values
+        :return dict: Value of environment variables, if provided. None otherwise
+        :raises IOError: If the file was not found or not a valid JSON
+        """
+        if not filename:
+            return None
+
+        # Try to read the file and parse it as JSON
+        try:
+
+            with open(filename, "r") as fp:
+                return json.load(fp)
+
+        except Exception as ex:
+            raise IOError(
+                "Could not read environment variables overrides from file {}: {}".format(filename, str(ex))
+            ) from ex
+
+    @staticmethod
+    def _make_env_vars(function: Function, env_vars_values: dict) -> Any:
```

Comment:
same here, `Any` -> `Dict`

Reply:
Dict wouldn't work as it can return none as well